MachObfuscatorTests/SymbolManglingTests/RealWordsMangler_Tests.swift

```diff
@@ -1,5 +1,55 @@
 import XCTest
 
 class RealWordsMangler_Tests: XCTestCase {
-    // TODO: test missing
+    // TODO: tests missing
+
+    var sentenceGenerator: ArraySentenceGenerator!
+    var sut: RealWordsMangler!
+
+    override func setUp() {
+        super.setUp()
+
+        sentenceGenerator = ArraySentenceGenerator()
+        sentenceGenerator.sentences = [
+            "bla1",
+            "bla2",
+            "bla3",
+            "blaBla1",
+            "blaBla2",
+            "blaBla3",
+        ]
+
+        sut = RealWordsMangler()
+    }
+
+    override func tearDown() {
+        sentenceGenerator = nil
+        sut = nil
+
+        super.tearDown()
+    }
+
+    private func when(symbols: ObfuscationSymbols) -> SymbolManglingMap {
+        return sut.mangleSymbols(symbols, sentenceGenerator: sentenceGenerator)
+    }
+
+    func test_mangleSymbols_shouldMangleSettersAndGettersCoherently() {
+        // Given
+        let whitelist = ObjCSymbols(selectors: [ "user", "view", "setUser:", "setView:" ], classes: [])
+        let blacklist = ObjCSymbols(selectors: [ "" ], classes: [])
```

Comment:
Perhaps it would make sense to add some selectors to blacklist and show what happens if they do and don't crash with the whitelist selectors? What do you think about it?

Reply:
Great idea! I have added this as another test case, so each test case name describes one requirement.